### PR TITLE
Add missing env vars needed for GPT4v deployment

### PR DESCRIPTION
**Pull request by pamelafox**
## Purpose

This PR adds missing azd output variables that are needed for CI/CD that has GPT-4-vision enabled.

## Does this introduce a breaking change?

```
[ ] Yes
[X] No
```

## Pull Request Type
What kind of change does this Pull Request introduce?

```
[X] Bugfix
[ ] Feature
[ ] Code style update (formatting, local variables)
[ ] Refactoring (no functional changes, no api changes)
[ ] Documentation content changes
[ ] Other... Please describe:
```

## How to Test

*  Set up `azd pipeline config` after setting up a gpt-4v deployment.

### CODE REVIEW

**Review by vhvb1989 — APPROVED** [review]
I assume that those variables are already defined in bicep and mapped in main.parameters.json. LGTM

**Comment by pamelafox** [comment]
Ooh, I realize our PRs now require approval by reviewers with write access. Fancy! Added Matt.

**Review by mattgotteiner — APPROVED** [review]

**Pull request merged**
